src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -1026,7 +1039,8 @@ namespace AppInstaller::Repository
                 AICLI_LOG(Repo, Info, << "Named source no longer found. Source was tombstoned: " << details.Name);

```

Comment:
I know this isn't part of your change, but I now realize that this results in the ability to remove the store source despite the fact that we want the well known sources to *always* be present.  I would suggest removing the Tombstone block.